homeassistant/components/device_tracker/__init__.py

```diff
@@ -89,7 +90,7 @@ def is_on(hass: HomeAssistantType, entity_id: str=None):
 def see(hass: HomeAssistantType, mac: str=None, dev_id: str=None,
         host_name: str=None, location_name: str=None,
         gps: GPSType=None, gps_accuracy=None,
-        battery=None):  # pylint: disable=too-many-arguments
+        battery=None, fuel_level=None):  # pylint: disable=too-many-arguments
```

Comment:
I don't like adding another parameter, it doesn't scale. I prefer if we just add an attributes dictionary as a parameter, that way any device tracker can add anything.